zkevm-circuits/src/evm_circuit.rs

```diff
@@ -14,8 +14,9 @@ pub(crate) mod util;
 pub mod table;
 
 use crate::table::{BlockTable, BytecodeTable, CopyTable, ExpTable, KeccakTable, RwTable, TxTable};
-use crate::util::{Challenges, SubCircuit, SubCircuitConfig};
+use crate::util::{log2_ceil, Challenges, SubCircuit, SubCircuitConfig};
```

Comment:
I think `log2_ceil` is a `test`-only import. So we could import it only in the tests mod.

Reply:
True!! I missread it then! My bad.